model/distro/distro.go

```diff
@@ -214,8 +214,10 @@ type HostAllocatorSettings struct {
 	MinimumHosts           int           `bson:"minimum_hosts" json:"minimum_hosts" mapstructure:"minimum_hosts"`
 	MaximumHosts           int           `bson:"maximum_hosts" json:"maximum_hosts" mapstructure:"maximum_hosts"`
 	RoundingRule           string        `bson:"rounding_rule" json:"rounding_rule" mapstructure:"rounding_rule"`
+	FeedbackRule           string        `bson:"feedback_rule" json:"feedback_rule" mapstructure:"feedback_rule"`
 	AcceptableHostIdleTime time.Duration `bson:"acceptable_host_idle_time" json:"acceptable_host_idle_time" mapstructure:"acceptable_host_idle_time"`
 	FutureHostFraction     float64       `bson:"future_host_fraction" json:"future_host_fraction" mapstructure:"future_host_fraction"`
+	WaitsFeedback          bool          `bson:"waits_feedback" json:"waits_feedback" mapstructure:"waits_feedback"`
```

Comment:
I think its better to use more words to be more descriptive; "waits feedback" doesn't really mean anything. Waits For Feedback? Or else I'd say drop some comments so it's clearer what this means.

Reply:
This was cruft. thanks for the catch / sorry for the mess

Reply:
Had to google "cruft" 😁

Reply:
I think this comment is still relevant to the HostAllocatorWaitsFeedback definition, however.